accurest-core/src/main/groovy/io/codearte/accurest/util/ContentUtils.groovy

```diff
@@ -166,18 +173,28 @@ class ContentUtils {
 		MapConverter.transformValues(parsedJson, { Object value ->
 			if (value instanceof String) {
 				String string = (String) value
-				Matcher matcher = TEMPORARY_PATTERN_HOLDER.matcher(string.trim())
-				if (matcher.matches()) {
-					List val = matcher[0] as List
-					String pattern = val[1]
-					return Pattern.compile(pattern)
-				}
-				return value
+				return returnParsedObject(string)
 			}
 			return value
 		})
 	}
 
+	static Object returnParsedObject(String string) {
+		Matcher matcher = TEMPORARY_PATTERN_HOLDER.matcher(string.trim())
+		if (matcher.matches()) {
+			List val = matcher[0] as List
+			String pattern = val[1]
```

Comment:
Think we should check if val contains 2 elements. If not some descriptive exception will be useful